client/src/helpers/APICalls/createProfile.ts

```diff
@@ -0,0 +1,62 @@
+import { FormikHelpers } from 'formik';
+
+interface FetchOptions {
+  method: string;
+  headers?: {
+    'Content-Type': string;
+  };
+  body?: string;
+  credentials?: RequestCredentials;
+}
+
+export interface IProfile {
+  firstName: string;
+  lastName: string;
+  email: string;
+  phone: string;
+  address: string;
+  gender: string;
+  birth: string;
+  description: string;
+}
+export interface Props {
+  handleSubmit?: (
+    { firstName, lastName, email, phone, address, gender, birth, description }: IProfile,
+    { setStatus, setSubmitting }: FormikHelpers<IProfile>,
+  ) => void;
+  validationSchema: { any: unknown };
```

Comment:
I didn't get this line. Could you explain?

Reply:
@ashrafcoding Please address above comment when possible

Reply:
@ashrafcoding Please see above